tests/test_algorithm.py

```diff
@@ -101,19 +114,19 @@ def test_cov_nearest(rng, mocker):
     cov = glass.algorithm.cov_nearest(a)
 
     # make sure all eigenvalues are positive
-    assert np.all(np.linalg.eigvalsh(cov) >= 0)
+    assert xp.all(xp.linalg.eigvalsh(cov) >= 0)
 
     # get normalisation
-    sq_d = np.sqrt(a.diagonal())
-    norm = np.outer(sq_d, sq_d)
+    sq_d = xp.sqrt(xp.linalg.diagonal(a))
+    norm = xp.linalg.outer(sq_d, sq_d)
 
     # make sure nearcorr was called with correct input
     nearcorr.assert_called_once()
     np.testing.assert_array_almost_equal_nulp(
         nearcorr.call_args_list[0].args[0],
-        np.divide(a, norm, where=(norm > 0), out=np.zeros_like(a)),
+        xp.divide(a, norm),
```

Comment:
Just checking why dropping `where`/`out`?

Reply:
`array_api_strict.divide` does not support those arguments. `out` is not testing anything here and the `where` argument is probably just reducing the number of computations (which is not very large in this case).